doc/conf.py

```diff
@@ -21,28 +35,59 @@
     "replacements",
     "smartquotes",
     "tasklist",
+    "colon_fence",
 ]
 
-exclude_patterns = [
-    "README.md",
-    "_build",
-]
-source_suffix = {
-    ".md": "markdown",
-}
+myst_dmath_double_inline = True
+myst_heading_anchors = 3
+
+# -----------------------------------------------------------------------------
+# BibTeX
+# -----------------------------------------------------------------------------
 
-# https://pypi.org/project/sphinxcontrib-bibtex/
 bibtex_bibfiles = ["references.bib"]
 bibtex_default_style = "unsrt"
 
-myst_dmath_double_inline = True
-myst_heading_anchors = 3
+# -----------------------------------------------------------------------------
+# Sphinx-book-theme
+# -----------------------------------------------------------------------------
 
 html_theme = "sphinx_book_theme"
 html_title = f"Spglib v{release}"
 html_theme_options = {
     # https://sphinx-book-theme.readthedocs.io/en/latest/reference.html
+    "repository_url": "https://github.com/spglib/spglib",
+    "use_repository_button": True,
     "navigation_with_keys": True,
     "globaltoc_includehidden": "true",
+    "show_toc_level": 3,
 }
 html_static_path = ["_static"]
+
+# -----------------------------------------------------------------------------
+# Autodoc2
+# -----------------------------------------------------------------------------
+autodoc2_output_dir = "python-interface/python-apidocs"
+autodoc2_render_plugin = "myst"
+autodoc2_packages = [
+    {
+        "path": "../python/spglib/spglib.py",
+        "module": "spglib",
+    }
+]
+autodoc2_docstring_parser_regexes = [
+    (r".*", 'rst'),
+]
```

Comment:
In default, no parser is selected. So I think this still needs.
https://sphinx-autodoc2.readthedocs.io/en/latest/config.html#confval-autodoc2_docstring_parser_regexes

Reply:
By default it should use whichever parser was used in the parent document, unless it is being overwritten, which I think that is what the option is for. Seem the more appropriate option to change is `autodoc2_render_plugin`

> Alternatively, you can set the [`autodoc2_render_plugin`](https://sphinx-autodoc2.readthedocs.io/en/latest/config.html#confval-autodoc2_render_plugin) configuration option in your conf.py:
> 
> ```
> autodoc2_render_plugin = "myst"
> ```
>
> This will now create all files with the “.md” extension, and thus the docstrings will be interpreted as MyST by default.